client-tags/message-status.md

```diff
@@ -0,0 +1,83 @@
+---
+title: IRCv3 `message-status` client tags
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+---
+## Purpose
+This specification defines several `message-status` client tags, which allows clients to send and receive various information about the status of messages before and after they are sent.
+
```

Comment:
`message-status` is just a name, rather than a CAP or a tag so shouldn't be code formatted.